newIDE/app/src/ObjectEditor/Editors/Model3DEditor.js

```diff
@@ -57,6 +59,18 @@ const styles = {
   },
 };
 
+const removeTrailingZeroes = (value: string) => {
```

Comment:
Check if it works with `0` or `0.0`?

Reply:
Yes, it does because it only removes `\.0+`